### PR TITLE
TO-DO website list structure

### PR DESCRIPTION
**Pull request by unorjikingsley**
In this project, I was able to :

1. Setup webpack for the project
2. Created index.html, index.js, and style.css files
3. Set an array of some simple to-do tasks (array of objects).
4. Created a function to iterate over the tasks array and populate an HTML list item element for each task.
5. Load CSS by Webpack Style/CSS Loader
6. Create source files that will be generated by Webpack and served by the Webpack dev server from the /dist folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,133 @@
...
+
+I would like to thank Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/unorjikingsley/To-Do-List6/blob/main/LICENSE) licensed.
```
Comment (Tufahel) [inline README.md:133]: - [ ] This link is not working perfectly. Kindly add a `MIT.md` file and link it here. Here is a [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) file recommended by microverse. `MIT.md` is a license file that assures you are the owner of a specific project. So it is professional to add every project you own.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @unorjikingsley :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Professional pull request
✔️ Correct git-flow
✔️ Linter checks are passing

## Required Changes ♻️

- [ ] The items which are showing in the `to-do-list` app are not similar to the items you have added in the `index.js` file.
- image
- I think you have forgotten to run `npm run build` after updating your code.
- Kindly consider my suggestion and check if it is working fine.

***Please kindly check the comments below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @unorjikingsley ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] Kindly remove the live demo section from your readme file since this project is not deployed. This is a general requirement for all projects.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @unorjikingsley , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**